Sources/KituraContracts/CodableQuery/Coder.swift

```diff
@@ -16,21 +16,44 @@
 
 import Foundation
 
-/// Class defining shared resources for the QueryDecoder and QueryEncoder
+/**
+ Class defining shared resources for the QueryDecoder and QueryEncoder.
```

Comment:
It would be nice if the words QueryDecoder and QueryEncoder linked to those classes.